app/templates/suppliers/duns_number.html

```diff
@@ -18,6 +18,17 @@
 
 {% block main_content %}
 
+{% if form.duns_number.errors[0] == 'DUNS number already used' %}
+<div class="validation-masthead" role="group" aria-labelledby="validation-masthead-heading">
+    <h1 class="validation-masthead-heading" id="validation-masthead-heading">
+        A supplier account already exists with that DUNS number
```

Comment:
Can we set this text in the actual form error instead of comparing the message only to rewrite it?